workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/README.md

```diff
@@ -0,0 +1,184 @@
+# Multi Source Security Viewer
+
+This plugin enables you to visualize pipeline security information from multiple sources.
+
+## Setting up the plugin
+
+### For adminstrators
+
+#### Prerequisites
+
+1. Install the CI providers you want the plugin to handle the security scan outputs from.
+   The supported CI's are:
+
+- [Jenkins](../../../jenkins/plugins/jenkins-backend/README.md)
+- [Github Actions](../../../github-actions/plugins/github-actions/README.md)
+- [Gitlab CI](https://github.com/immobiliare/backstage-plugin-gitlab?tab=readme-ov-file#setup)
+
+#### Permission Framework Support
+
+The MSSV plugin has support for the permission framework.
+
+- When [RBAC permission](https://github.com/backstage/community-plugins/tree/main/workspaces/rbac/plugins/rbac-backend#installation) framework is enabled, for non-admin users to access MSSV UI, the role associated with your user should have the following permission policies associated with it.
```

Comment:
Using this plugin isn't the only way to use permissions, I would include and example PermissionPolicy in this documentation as well. 👍

Reply:
Sure, example added.